### ISSUE TITLE
User Review Checklist

### ISSUE THREAD

**Issue by justineechen** [OpenLiberty/draft-guide-cloud-aws#11]
User Review<br/>- [ ] Record the time it takes to go through the entire guide<br/>- [ ] Test each step in the guide is clear to understand<br/>- [ ] Ensure no logic errors<br/>- [ ] Resolve anything that does not work<br/>- [ ] Resolve anything usability issues (confusing or hard to follow)<br/>- [ ] Achieve basic understanding/working knowledge of the topic in the guide

**Comment by andrewdes** [comment]
Closing since User Review feedback is listed here #26
